src/Chainweb/Version.hs

```diff
@@ -578,21 +580,22 @@ txEnabledDate TimedCPM{} = Nothing
 txEnabledDate FastTimedCPM{} = Nothing
 txEnabledDate Development = Just [timeMicrosQQ| 2019-11-30T05:00:00.0 |]
 txEnabledDate Testnet04 = Nothing
-txEnabledDate Mainnet01 = Just [timeMicrosQQ| 2019-12-17T01:00:00.0 |]
+txEnabledDate Mainnet01 = Just [timeMicrosQQ| 2019-12-17T15:30:00.0 |]
 
 -- | KILLSWITCH: The date after which nodes in the 1.1.x series will
 -- spontaneously allow Transactions in the system. This constant can be removed
 -- once the date has passed, and /must not be used in core validation code/.
+-- Must be after 'txEnabledDate' in mainnet.
 --
 transferActivationDate :: ChainwebVersion -> Maybe (Time Micros)
 transferActivationDate Test{} = Nothing
 transferActivationDate TimedConsensus{} = Nothing
 transferActivationDate PowConsensus{} = Nothing
 transferActivationDate TimedCPM{} = Nothing
 transferActivationDate FastTimedCPM{} = Nothing
-transferActivationDate Development = Just [timeMicrosQQ| 2019-11-30T07:00:00.0 |]
+transferActivationDate Development = Just [timeMicrosQQ| 2019-12-13T23:25:00.0 |]
 transferActivationDate Testnet04 = Nothing
-transferActivationDate Mainnet01 = Just [timeMicrosQQ| 2019-12-17T01:00:00.0 |]
+transferActivationDate Mainnet01 = Just [timeMicrosQQ| 2019-12-17T16:00:00.0 |]
```

Comment:
I agree with this date.  We want this to be late morning NYC time so it will be a convenient time for us to be on top of any potential issues.